package.json

```diff
@@ -56,26 +56,30 @@
     "@floating-ui/dom": "^1.1.0",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
-    "@zestia/ember-auto-focus": "^4.2.0",
+    "@zestia/ember-auto-focus": "^4.2.0 || ^5.0.0",
```

Comment:
I'm not sure if this is breaking or not 🤔. My intention was that apps that _do_ want v5, can install it, and Appuniversum would follow (if deduped), but not every app will have this installed. I'm not how the package managers will behave either when they see this range. They could in theory install the latest version.